Network-Security/Integration/aws-python-amazon-inspector-integration/c1ns-inspector-findings.yml

```diff
@@ -0,0 +1,957 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
Following what we discussed about having an S3 bucket to store the code, it would be a good idea to have the py code in a separated file and attach the code to the lambda, so we can easily upgrade the code, lint and find bugs.

Reply:
Do you mean putting the actual py file that was used in the Lambda function in addition to the CFT file?